chrome_extensions/Pedrokeys/hiq_hotkeys.js

```diff
@@ -30,15 +30,29 @@ const BANNER_CLASS = 'hiq-banner';
 const HANGUP_BUTTON = '.end-contact';
 const HANGUP_CONFIRM_BUTTON = '.confirm-end-contact';
 const SUBMIT_BUTTON = '.save-close-acw-submit';
-const PREFIX = 'ctrl';
-const FRIENDLY_PREFIX = 'Ctrl';
+var PREFIX = "";
```

Comment:
if it's a var, don't make it in all caps. That's only for constants. Also, don't have global variables. You can just remove this part here.